src/WebSocketStream.h

```diff
@@ -150,7 +150,7 @@ namespace aws {
                  * @param handler the callback handler when the async operation is complete.
                  */
                 void
-                async_ssl_handshake(const ssl::stream_base::handshake_type &type, const BoostCallbackFunc &handler);
+                async_ssl_handshake(const ssl::stream_base::handshake_type &type, const std::string &host, const BoostCallbackFunc &handler);
```

Comment:
Could you please rename it to `web_proxy_host` and also add it as a param on line 150? We used `host` [here](https://github.com/aws-samples/aws-iot-securetunneling-localproxy/blob/b6c31b442fa6d4c930e705d2853efd73dadff169/src/WebSocketStream.h#L164C22-L164C40) to refer the websocket host/endpoint.